python/TestHarness/tests/test_FailedJSON.py

```diff
@@ -1,51 +0,0 @@
-#* This file is part of the MOOSE framework
-#* https://www.mooseframework.org
-#*
-#* All rights reserved, see COPYRIGHT for full restrictions
-#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
-#*
-#* Licensed under LGPL 2.1, please see LICENSE for details
-#* https://www.gnu.org/licenses/lgpl-2.1.html
-
-import os, sys, io
-import unittest
-import mock
-import TestHarness
-from contextlib import redirect_stdout
-
-class TestHarnessTester(unittest.TestCase):
```

Comment:
We should not remove this

Reply:
Re-added to double check what this actually is, and this is actually checking the --json output on the testharness, so yes, good catch. I will readd with the rebase.